iac-extensions/docker/src/main/java/org/sonar/iac/docker/checks/DirectoryCopySourceCheck.java

```diff
@@ -43,12 +45,12 @@ public class DirectoryCopySourceCheck implements IacCheck {
   @Override
   public void initialize(InitContext init) {
     init.register(AddInstruction.class, DirectoryCopySourceCheck::checkAdd);
-    init.register(CopyInstruction.class, DirectoryCopySourceCheck::checkCopy);
+    init.register(CopyInstruction.class, ignoringHeredoc(DirectoryCopySourceCheck::checkCopy));
```

Comment:
Please add test case from ticket for this change

Reply:
Good catch!